### PR TITLE
Integration tests for IDE / templates package management

### PR DESCRIPTION
**Pull request by vlada-shubina**
fixes https://github.com/dotnet/templating/issues/2920
Integration tests for IDE / templates package management

### CODE REVIEW

File: src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs [hunk 1]
```diff
@@ -88,41 +88,41 @@ public async Task<ICreationEffects> GetCreationEffectsAsync(ITemplateInfo info,
...
-        /// - the folder containing the template
+        /// - the folder containing the template.
         /// </summary>
-        /// <param name="installRequests">the list of <see cref="InstallRequest"/> to install</param>
-        /// <param name="scope"><see cref="InstallationScope"/> to use</param>
+        /// <param name="installRequests">the list of <see cref="InstallRequest"/> to install.</param>
+        /// <param name="scope"><see cref="InstallationScope"/> to use.</param>
         /// <param name="cancellationToken"></param>
-        /// <returns>the list of <see cref="InstallResult"/> containing installation result for each <see cref="InstallRequest"/></returns>
-        public Task<IReadOnlyList<InstallResult>> InstallAsync(IEnumerable<InstallRequest> installRequests, InstallationScope scope = InstallationScope.Global, CancellationToken cancellationToken = default)
```
Comment (DavidKarlas) [inline src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs:125]: `Obsolete` messages in this class need to be updated to reflect this change.

**Review by DavidKarlas — APPROVED** [review]
Looks OK. Just minor comment about Obsolete text.

File: src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs [hunk 2]
```diff
@@ -88,41 +88,41 @@ public async Task<ICreationEffects> GetCreationEffectsAsync(ITemplateInfo info,
         #region Template Package Management
 
         /// <summary>
-        /// Get the list of available template packages
+        /// Gets the list of available templates packages.
         /// </summary>
         /// <param name="cancellationToken"></param>
-        /// <returns>the list of the template packages</returns>
-        public Task<IReadOnlyList<ITemplatesSource>> GetTemplatesSources(CancellationToken cancellationToken = default)
+        /// <returns>the list of the templates packages.</returns>
```
Comment (bozturkMSFT) [inline src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs:94]: This plural format looks wrong to me in any example I see. I say we revert this and go with "template packages".

**Review by bozturkMSFT — COMMENTED** [review]
Tests look good to me, but "templates packages" just sounds wrong.

File: src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs (hunk 2 above)
Comment (vlada-shubina) [reply src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs:94]: That's a confusion that I have with `TemplatesPackage`. How to do plural from it?

Comment (vlada-shubina) [reply src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs]: Reverted to use "template package(s)"

**Review by bozturkMSFT — APPROVED** [review]

**Pull request merged**